src/man/sssd.conf.5.xml

```diff
@@ -2215,6 +2229,75 @@ pam_gssapi_indicators_map = sudo:pkinit, sudo-i:pkinit
                         </para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term>pac_check (string)</term>
+                    <listitem>
+                        <para>
+                            Apply additional checks on the PAC of the Kerberos
+                            ticket which is available in Active Directory and
+                            FreeIPA domains, if configured. The following
+                            options are available:
```

Comment:
I would mention multiple options can be enumerated using ',' delimiter. Or maybe it's enough to specify type as a 'string list'.